osu.Game/OsuGame.cs

```diff
@@ -220,6 +220,12 @@ public partial class OsuGame : OsuGameBase, IKeyBindingHandler<GlobalAction>, IL
 
         private readonly List<OverlayContainer> visibleBlockingOverlays = new List<OverlayContainer>();
 
+        /// <summary>
+        /// Whether the game should be limited from providing access to download non-featured-artist beatmaps.
+        /// This only affects the "featured artists" filter in the beatmap listing overlay.
+        /// </summary>
+        public bool LimitedToFeaturedArtists => RuntimeInfo.OS == RuntimeInfo.Platform.iOS;
```

Comment:
```suggestion
        public bool LimitedToFeaturedArtists => IsDeployedBuild && RuntimeInfo.OS == RuntimeInfo.Platform.iOS;
```

Hmm, you may want to only limit this to a published build. If someone reports an error in a specific non-featured beatmap on iOS, you'll want to be able to download it.

Reply:
Unnecessary IMO. You can add this yourself if required.

Reply:
Ummmm, that's actually what I had in my branch before replacing it with a `true` to test that it works :eyes:. Doesn't matter that much to me either way.